include/votca/xtp/anderson_mixing.h

```diff
@@ -0,0 +1,53 @@
+/*
+ *            Copyright 2009-2020 The VOTCA Development Team
+ *                       (http://www.votca.org)
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License")
+ *
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#pragma once
+#ifndef _VOTCA_XTP_ANDERSON__H
+#define _VOTCA_XTP_ANDERSON__H
+
+#include <iostream>
+#include <memory>
+#include <vector>
+#include <votca/xtp/eigen.h>
+namespace votca {
+namespace xtp {
+
+class ANDERSON {
+ public:
+  Eigen::VectorXd NPAndersonMixing(const double alpha);
```

Comment:
what does NP stand for?

Reply:
const double alpha is not necessary. double will do. What does it return?

Reply:
alpha should be past in the set order/configure method.

Reply:
and as this is an action, please turn the function name into a verb.

Reply:
Will create a Configure function in place of SetOrder